lms/templates/static_templates/careers.html

```diff
@@ -9,7 +9,7 @@
       <div class="white-panel">
         <div class="row">
           <div class="col-md-9">
-            ${urllib2.urlopen('https://gymnasium.github.io/static-site-content/careers.html').read()}
+            ${gymcms.render('careers')}
```

Comment:
This template gives a 404 for both staging and prod: 

- https://staging.gymcms.xyz/static/careers
- https://gymcms.xyz/static/careers

Reply:
Good catch - updating this.